subprojects/launcher/src/integTest/groovy/org/gradle/launcher/daemon/AbstractDaemonLifecycleSpec.groovy

```diff
@@ -72,13 +72,15 @@ class AbstractDaemonLifecycleSpec extends DaemonIntegrationSpec {
                 executer.withDefaultCharacterEncoding(buildEncoding)
             }
             executer.usingProjectDirectory buildDirWithScript(builds.size(), """
+                def stopFile = file("stop")
+                def existsFile = file("exists")
                 task('watch') {
                     doLast {
                         println "waiting for stop file"
                         long sanityCheck = System.currentTimeMillis() + 120000L
-                        while(!file("stop").exists()) {
```

Comment:
💭 Any idea why this didn't fail without load after store? Since the failures have been ignored?

Reply:
Not sure I understand your question, Stefan. Before load after store, for the first run (a cache miss), CC-enabled behaves a lot like non-CC. In order to see CC-specific behavior, you would need a cache hit (i.e. a second run).

Reply:
I'd have expected that this gives a configuration cache warning/error on store, since there is access to the project in the task action.